### DISCUSSION TITLE
Google Collab VS PaperSpace

### DISCUSSION THREAD

**Discussion by dymension** [TheLastBen/fast-stable-diffusion#2776] [General]
Hello Ben
in one of your answers to [darth2602] you're telling him

> go for RunPod or paperspace, it'll save you a lot of time and pain

Ok so I tried paperspace, and, after finding out (in the Discussion) that the correct URL to use was **_https://github.com/TheLastBen/PPS_**, and after subscribing to a paying Machine (since all free machine are not available), I could manage to Launch A1111 on PaperSpace

but I cannot generate any picture, because I'm instantly out of memory
`RuntimeError: CUDA error: CUBLAS_STATUS_NOT_INITIALIZED when calling 'cublasCreate(handle)' `
`A: 6.72 GB, R: 6.99 GB, Sys: 7.8/7.78809 GB (100.0%)`

Still, I'm using a machine with 30Gb RAM and 8Gb GPU, I can't believe I'm out of memory for a mere 512x512 picture 1 shot. Is there more needs ine PaperSpace than in Google Collab ?

So it looks more expesive than Collab at the end .

how can you reduce the GPU consomption ?

**Comment by dymension** [comment]
So I made some more test
Google Collab Costs 10 € / month and offer 100 hours (let's call it 100 units)
Since I test and use a  lot of SD model, I need 110 Go to have them all (15 models), so I pay the 200 Go of Google Drive, that is 3 €/month
so 
**Google Collab, 200 Go + 100 units = 13 € / month**

In PaperSpace, I need to pay for the Pro status, which is 8 €, but then I have to use a machine with at least 16 Go GPU or no image will be created, (CUDA error) which cost 0.70/h. So for 100h, it will cost me 70 €
Then paperspace only let me use 15 Go. Each 1Gb more is 0.30 $ / Go, so 200 Go will cost 55 $/ month
so
**paperSpace 200Go + 100 units = 125 € / month**

Really Paperscpae is way too expensive for us to move to this platform

So Thank You if you can still make it work on Google Collab

**Reply by TheLastBen** [reply]
Try RunPod, you can get gpus as low as 0.14$/hour

**Reply by dymension** [reply]
Well, the cheapest I find is 0.28 $ /h if I pay 6 month. i'd rather use per month which is 0.32 $
200 Go will cost 14 $ (110 will be 8$), which is not bad
So
**RunPod  200Go + 100 units = 46 $ / month**

(but even 0.14 $ /h will be more expensive than Collab : 14$+ 8 $ at least = 22 $ /m)

Still, Collab is the Cheapest

**Reply by TheLastBen** [reply]
choose community servers

**Reply by dymension** [reply]
I can't find that
But if it is the FREE server, it's never available
